serving/samples/thumbnailer-go/README.md

```diff
@@ -1,44 +1,55 @@
 # Thumbnailer Demo
 
-Thumbnailer demo is a walk-through example on how to deploy a 'dockerized' application to the Knative Serving service. In this demo we will use a sample `golang` application that takes video URL as an input and generates its thumbnail image.
+This is a walk-through example that demonstrates deploying a dockerized
+application to the Knative Serving service. In this demo we will use a sample
+`golang` application that takes a video URL as an input and generates its thumbnail image.
 
-> In this demo we will assume access to existing Knative Serving service. If not, consult [README.md](https://github.com/knative/serving/blob/master/README.md) on how to deploy one.
+## Before you begin
 
-## Sample Code
+* [Install Knative Serving](../../../install/README.md)
+* [Install Go](https://golang.org/doc/install) (if you want to run and test the sample app locally)
 
-In this demo we are going to use a simple `golang` REST app called [rester-tester](https://github.com/mchmarny/rester-tester). It's important to point out that this application doesn't use any 'special' Knative Serving components nor does it have any Knative Serving SDK dependencies.
+## Sample code
 
-### App code
+In this demo we are going to use a simple `golang` REST app called
+[rester-tester](https://github.com/mchmarny/rester-tester). It's important
```

Comment:
So there's a lot going on in Rester Tester: https://github.com/mchmarny/rester-tester

I'm not 100% what to copy into this sample repo and what to leave behind. For example, if I were to copy all the files, would I want to bring along the .gcloudignore, .gitignore, and .travis.yml files? And rester-tester also includes the ability to calculate prime numbers; could I leave that folder out since this sample doesn't use that functionality? (Looks like that might require a minor code re-write...) And would we just leave out the original README, in favor of this one? I guess the other option is to keep everything in a subfolder of /thumbnailer-go/...